aptos-move/framework/aptos-framework/sources/IterableTable.move

```diff
@@ -73,15 +73,6 @@ module AptosFramework::IterableTable {
         &mut Table::borrow_mut(&mut table.inner, key).val
     }
 
-    /// Acquire a mutable reference to the value which `key` maps to.
```

Comment:
Seems like you accidentally removed this function in your latest commit?

Reply:
You're correct, I brought it back